src/Routers/UsersRouter.js

```diff
@@ -25,7 +26,20 @@ export class UsersRouter extends ClassesRouter {
     let data = deepcopy(req.body);
     req.body = data;
     req.params.className = '_User';
-    return super.handleCreate(req);
+
+    if (req.config.verifyUserEmails) {
```

Comment:
I'm fine leaving it like this for now. My motto is to write easy-to-fix code, not perfect code :p